lighthouse-core/gather/gatherers/image-usage.js

```diff
@@ -35,15 +35,23 @@ function collectImageElementInfo() {
     const entries = srcset.split(',');
     const relativeUrls = entries.map(entry => entry.trim().split(' ')[0]);
     return relativeUrls.map(url => {
-      const a = document.createElement('a');
-      a.href = url;
-      return a.href;
+      try {
+        return new URL(url, window.location.href).href;
```

Comment:
If the site uses a `<base href="http://somethingelse.com/">`, `new URL(url, window.location.href).href;` might not be correct. e.g. http://jsbin.com/yilakeqeyi/edit?html,output

`return new URL(url, document.baseURI || window.location.href).href;` should do it.